src/components/fieldLabel/fieldLabel.tsx

```diff
@@ -10,8 +10,13 @@ interface FieldLabelProps
   isRequired?: boolean;
 }
 
-export const FieldLabel: FC<FieldLabelProps> = ({ children, isRequired = false, ...rest }) => (
-  <label className={cx('field-label')} {...rest}>
+export const FieldLabel: FC<FieldLabelProps> = ({
```

Comment:
@AmsterGet do you agree to accept no FC type rule for ui kit?
https://github.com/reportportal/service-ui/blob/develop/app/TYPESCRIPT_MIGRATION_GUIDE.md#why-not-use-fc-functioncomponent

Reply:
@Guria sure.
Let's get rid of it gradually

Reply:
@Guria @AmsterGet removed FC for the changed components